### PR TITLE
this.logging

### PR DESCRIPTION
**Pull request by Andriy2**
Hide log messages from logging

If I have my own beautiful logger, I don't need redundant messages in my log (also we have events "start", "stop", "error", ...)

### CODE REVIEW

File: lib/telebot.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ class TeleBot {
         this.usePlugins = Array.isArray(cfg.usePlugins) ? cfg.usePlugins : [];
         this.defaultPlugins = cfg.defaultPlugins !== undefined ? (cfg.defaultPlugins || []) : DEFAULT_PLUGINS;
         this.pluginConfig = cfg.pluginConfig || {};
+        this.logging = cfg.logging || true;
```
Comment (mullwar) [inline lib/telebot.js]: Incorrect comparison `this.logging` always will be `true`.

Comment (Andriy2) [inline lib/telebot.js]: Oops...

**Comment by Andriy2** [comment]
Please, check now

File: lib/telebot.js [hunk 2]
```diff
@@ -93,10 +94,14 @@ class TeleBot {
 
             plugin.call(this, this, config || {});
 
-            console.log(`[bot.plugin] loaded '${id}' plugin`);
+            if(this.logging) {
```
Comment (mullwar) [inline lib/telebot.js]: Add space before `if` statements.
```js
if () { ... }
```

**Pull request merged**